server/text_generation_server/models/custom_modeling/flash_llama_modeling.py

```diff
@@ -470,12 +492,15 @@ class FlashLlamaForCausalLM(torch.nn.Module):
     def __init__(self, prefix: str, config, weights):
         super().__init__()
 
-        self.embed_tokens = TensorParallelEmbedding(
-            prefix=(
-                "model.embed_tokens" if not prefix else f"{prefix}.model.embed_tokens"
-            ),
-            weights=weights,
-        )
+        with no_fp8(weights):
```

Comment:
I think you are also missing the `lm_head`? or is not necessary?

Reply:
Added.